client/src/components/Header.css

```diff
@@ -0,0 +1,34 @@
+.header { 
+    z-index: 3;
+    height: 70px;
+    width: 100%;
+    margin: 0px auto;
+    position: fixed;
+    background: rgba(0, 0, 0, 0.53);
+    transition: all 0.2s ease-in-out;
+}
+
+#text-header{
+    padding-top: 20px;
+    color:white;
+    font-size: 20px;
+    text-decoration: none;
+}
+
+
+.header:hover {
+    background: rgb(0, 0, 0);
+    -webkit-transition: all 0.2s ease-in-out;
+    -moz-transition: all 0.2s ease-in-out;
+    -o-transition: all 0.2s ease-in-out;
+    transition: all 0.2s ease-in-out;
+}
+
+.brand-logo {
+    margin-left: 420px;
+}
+
+.right {
+    margin-right: 20px;
+    padding-top: 5px;
+}
```

Comment:
Won't this conflict with Materialize's "right" class? What if we want to use materialize's right class?

Also, margin right and padding top could be put inside the global.css.

Reply:
I'll look into materialize's right class and how to customize it. The right class currently moves the text 100% to the right, leaving no white space between the text and the end of the window.

Reply:
I would leave the Materialize class alone and make your own class named something different.

It's best to try not to override Materialize.